assets/sass/v2/_all.scss

```diff
@@ -23,6 +23,7 @@
   // the following is to center the beacon vertically with the divider line
   .siw-main-view {
     float: left; // to prevent margin collapse which drags beacon down
```

Comment:
are there other approaches instead of `float: left` the main container?

Reply:
+1 to this comment, can we explore another way to center the beacon vertically?

Reply:
https://github.com/okta/okta-signin-widget/pull/1854/commits/8ce08882f70a944c802acccae1a62a04ce5efe3b